core/imodel-bridge/src/BadgersIssueReporter.ts

```diff
@@ -0,0 +1,283 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import { DbResult, Guid, GuidString, Logger } from "@bentley/bentleyjs-core";
+import { ECDb } from "@bentley/imodeljs-backend";
+import { BridgeIssueReporter } from "./BridgeIssueReporter"
+import * as fs from "fs";
+import * as path from "path";
+import * as os from "os";
+import { BridgeLoggerCategory } from "./BridgeLoggerCategory";
+
+interface ContextInfo {
+  activityId: string;
+  contextId: string;
+  iModelId: string;
+  briefcaseId?: number;
+  operationType?: string;
+  jobId?: string;
+}
+
+interface FileInfo {
+  fileId?: string;
+  filePath?: string;
+}
+
+export interface PostPublishingProcessor {
+  (reportPath: string): Promise<void> | void;
+}
+
+/** BADGERS issue reporter. Can be provided to BridgeRunner in setIssueReporter
+ * @beta
+ */
+export class BadgersIssueReporter implements BridgeIssueReporter {
```

Comment:
We still want a better name... don't want to use "BADGERS". Maybe SqliteIssueReporter, since that's the crux of what it does?

Reply:
Should I go ahead and rename the tables in the schema as well? Do we need to keep the schema consistent with the native issue reporter?

Reply:
Let's keep the table names for now... that's an internal implementation detail we can more easily change later.